docs/installation/linux.md

```diff
@@ -443,12 +443,12 @@ chmod a+x PowerShell-6.0.0-alpha.18-x86_64.AppImage
 ### Uninstallation - Kali
```

Comment:
`alpha.18` is used in the `Kali` section, and they need to be updated to `beta.4`.